infra/ansible/e2e_tests.Dockerfile

```diff
@@ -1,8 +1,12 @@
 ARG python_version=3.8
-ARG debian_version=buster
+ARG debian_version=bullseye
 
 FROM python:${python_version}-${debian_version} AS build
 
+# install gcc10
+RUN apt-get update
+RUN apt-get install -y gcc-10
```

Comment:
Should gcc-10 be installed by ansible instead? This image installs all of the packages from the `build` stage.